addons/service/downloadmanager/FlexGet/source/bin/FlexGet.start

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
+
+################################################################################
+#      This file is part of OpenELEC - http://www.openelec.tv
+#      Copyright (C) 2009-2012 Stephan Raue (stephan@openelec.tv)
+#
+#  This Program is free software; you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation; either version 2, or (at your option)
+#  any later version.
+#
+#  This Program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with OpenELEC.tv; see the file COPYING.  If not, write to
+#  the Free Software Foundation, 51 Franklin Street, Suite 500, Boston, MA 02110, USA.
+#  http://www.gnu.org/copyleft/gpl.html
+################################################################################
+
+. /etc/profile
+
+oe_setup_addon service.downloadmanager.FlexGet
+
+if [ "$FLEXGET_PATH_CUSTOM_CONFIG" = "" ]; then
+  FLEXGET_PATH_CUSTOM_CONFIG="/storage/.config/flexget"
+fi
+mkdir -p $FLEXGET_PATH_CUSTOM_CONFIG
+
+# Addon settings
+ADDON_DIR="$HOME/.kodi/addons/service.downloadmanager.FlexGet"
+ADDON_HOME="$HOME/.kodi/userdata/addon_data/service.downloadmanager.FlexGet"
```

Comment:
ADDON_DIR and ADDON_HOME are exported from from oe_setup_addon(), no need to redefine